snapcraft/internal/pluginhandler/_runner.py

```diff
@@ -124,32 +138,27 @@ def _run_scriptlet(self, scriptlet_name: str, scriptlet: str,
             feedback_fifo = _NonBlockingRWFifo(
                 os.path.join(tempdir, 'call_feedback'))
 
-            env = ''
-            if common.is_snap():
-                # Since the snap is classic, $SNAP/bin is not on the $PATH.
-                # Let's set an alias to make sure it's found (but only if it
-                # exists).
-                snapcraftctl_path = os.path.join(
-                    os.getenv('SNAP'), 'bin', 'snapcraftctl')
-                if os.path.exists(snapcraftctl_path):
-                    env += 'alias snapcraftctl="$SNAP/bin/snapcraftctl"\n'
-            env += common.assemble_env()
-
             # snapcraftctl only works consistently if it's using the exact same
             # interpreter as that used by snapcraft itself, thus the definition
             # of SNAPCRAFT_INTERPRETER.
             script = textwrap.dedent("""\
                 export SNAPCRAFTCTL_CALL_FIFO={call_fifo}
                 export SNAPCRAFTCTL_FEEDBACK_FIFO={feedback_fifo}
                 export SNAPCRAFT_INTERPRETER={interpreter}
-                {env}
```

Comment:
I definitely understand and appreciate the extraction of the env stuff above into a method, but I don't understand why it was extracted from this script only to be added later to the script below. I'd kinda prefer to see the script all together in one place, personally.

Reply:
Oh, I wanted to split up the "real" script with the modifications we've done to the script, it becomes easier to debug at that point